llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -1150,15 +1150,21 @@ def Feature32Bit
     : SubtargetFeature<"32bit", "IsRV32", "true", "Implements RV32">;
 def Feature64Bit
     : SubtargetFeature<"64bit", "IsRV64", "true", "Implements RV64">;
+def FeatureEncodingTmp
```

Comment:
What's this for? Testing?

Reply:
Yes, this is just a reference and will be deleted eventually. It’s mainly to help everyone easily understand the purpose of this patch.